inventory.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think the `inventory.json` should be removed out of source control, since it is dynamically generated by code.